src/xenia/cpu/backend/x64/x64_function.cc

```diff
@@ -30,13 +30,12 @@ void X64Function::Setup(uint8_t* machine_code, size_t machine_code_length) {
   machine_code_length_ = machine_code_length;
 }
 
-bool X64Function::CallImpl(ThreadState* thread_state, uint32_t return_address) {
+void X64Function::CallImpl(ThreadState* thread_state, uint32_t return_address) {
   auto backend =
       reinterpret_cast<X64Backend*>(thread_state->processor()->backend());
   auto thunk = backend->host_to_guest_thunk();
   thunk(machine_code_, thread_state->context(),
         reinterpret_cast<void*>(uintptr_t(return_address)));
-  return true;
```

Comment:
While we don't currently do anything with the return value, that's left open for in case we want to in the future.

I really doubt this has an impact on performance, do you have any metrics on this specific change alone?

Reply:
no, it probably doesn't. I'll revert it